Renamed KeyPoint to Keypoint (#206)
Unity-Technologies/com.unity.perception: com.unity.perception/CHANGELOG.md

### Upgrade Notes

All appearances of the term `KeyPoint` have been renamed to `Keypoint`. Therefore, the following steps will be required after upgrading to this version:
* If you have code that relies on any renamed types or names, alter your code to reflect the new names.

> * If you are using custom `KeypointTemplate` assets besides the provided `CocoKeypointTemplate`, open those assets in a text editor and change the `keyPoints:` line to `keypoints:`, then save the file.

Reviewer: Can you use `[FormerlySerializedAs("keyPoints")]` to get this automatically?

Author: yes! will do.

### Changed

> Renamed all instances of "KeyPoint" to "Keypoint" for standards adherence.

Author: @JonathanHUnity I was having a hard time coming up with a good log line here. Please feel free to change this!

Reviewer 2: ```suggestion
Renamed all types and members named "KeyPoint" to "Keypoint"
```
How about something like this?

Reviewer 2: Also, we should add an upgrade note as well on changes they will have to make. It's going to be a pain for any current users.

Author: > How about something like this?

I like it. But technically, it is also other appearances of the term as part of other names, e.g. KeypointLabeler.
So I changed it to this:

Renamed all appearances of the term "KeyPoint" within types and names to "Keypoint".
